manifests/apache.pp

```diff
@@ -17,7 +16,7 @@
   $expiration           = $certs::expiration,
   $default_ca           = $certs::default_ca,
   $ca_key_password_file = $certs::ca_key_password_file,
-  $group                = $certs::group,
+  $group                = 'root',
```

Comment:
Apache actually reads certificates and keys as the root user so this is the tightest permission that will still work.